big_tests/tests/carboncopy_SUITE.erl

```diff
@@ -147,8 +146,7 @@ dropped_client_doesnt_create_duplicate_carbons(Config) ->
       fun(Alice1, Alice2, Bob) ->
               Msg = escalus_stanza:chat_to(Bob, <<"And pious action">>),
               given_client_logs_out(Config, Alice2),
-              _Pres = escalus_client:wait_for_stanza(Alice1),
-
+               _Pres = escalus_client:wait_for_stanza(Alice1),
```

Comment:
Couldn't we make the same thing here as we did [here](https://github.com/esl/MongooseIM/pull/1982/files#diff-391a6ab01d8b3ee0ba4f9a38fe015fe0R125)

Reply:
I mean, not exactly the same. I suggest that we either don't bind to `_Pres` here or check that it is a stanza "presence".